### PR TITLE
[CORL-2818] fix mod queue ban shortcut

### PR DESCRIPTION
**Pull request by marcushaddon**
## What does this PR do?
This PR addresses a regression in which the `b` key shortcut no longer opened the Ban Modal in the moderation queue.

## These changes will impact:

- [ ] commenters
- [ x] moderators
- [x ] admins
- [ ] developers

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags?
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As an admin or moderator navigate to the moderation queue
2. press the `b` key, and observe the the ban modal appears, and is for the user of the currently focused comment

## Where any tests migrated to React Testing Library?
No.

## How do we deploy this PR?
No special considerations needed.

### CODE REVIEW

File: src/core/client/admin/components/BanModal.tsx [hunk 1]
```diff
@@ -309,54 +313,137 @@ const BanModal: FunctionComponent<Props> = ({
...
-                          : rejectExistingCommentsLocalizationId!
-                      }
+                {/* MAIN */}
+                <Flex
+                  direction="column"
+                  className={styles.form}
+                  justifyContent="flex-start"
+                  spacing={3}
+                >
+                  {/* BAN FROM/REJECT COMMENTS */}
```
Comment (nick-funk) [inline src/core/client/admin/components/BanModal.tsx:323]: I'm seeing a lot of commented sections to this large file. Would it make sense to break them out into smaller named components?

File: src/core/common/permissions/types.ts [hunk 2]
```diff
@@ -71,17 +71,17 @@ export const isModerator = (user: Readonly<User>): user is Moderator => {
...
-export const isSiteModerator = (user: Readonly<User>): user is SiteModerator =>
+export const isSiteModerator = (user: User): user is SiteModerator =>
   user.role === "MODERATOR" && isSiteModerationScoped(user.moderationScopes);
 
-export const isOrgModerator = (user: Readonly<User>): user is OrgModerator =>
+export const isOrgModerator = (user: User): user is OrgModerator =>
   user.role === "MODERATOR" && !isSiteModerationScoped(user.moderationScopes);
 
-export const isLTEModerator = (user: Readonly<User>): user is LTEModerator => {
+export const isLTEModerator = (user: User): user is LTEModerator => {
```
Comment (nick-funk) [inline src/core/common/permissions/types.ts:80]: Just curious, what is an `LTEModerator` does LTE stand for `less than or equal to` or something similar?

File: src/core/client/admin/components/BanModal.css [hunk 3]
```diff
@@ -12,6 +12,31 @@ $ban-modal-text: var(--palette-text-500);
   height: calc(12 * var(--mini-unit));
 }
 
-.sitesToggle {
+.domainBanHeader {
+  margin-bottom: var(--spacing-2);
+
```
Comment (nick-funk) [inline src/core/client/admin/components/BanModal.css:17]: lol, looks like we need to take a peek at our CSS auto-formatter at some point, it's letting us leave errant spaces behind

**Review by nick-funk — APPROVED** [review]
This looks good, seems to work well!

Left some comments with random thoughts while perusing, nothing that would hold back an approve though.

Looks like the linter is unhappy, but after that's fixed up, should be good to merge!

File: src/core/client/admin/components/BanModal.tsx (hunk 1 above)
Comment (marcushaddon) [reply src/core/client/admin/components/BanModal.tsx:323]: That would probably be a good idea. I can do that, but this is from one of the commits I didn't mean to include. This is for the PR re: fixing the ban modal design

File: src/core/common/permissions/types.ts (hunk 2 above)
Comment (marcushaddon) [reply src/core/common/permissions/types.ts:80]: Yes, exactly, that would be worth a doc string

**Pull request closed without merging**